### ISSUE TITLE
Indentation rules incorrectly flag properly indented code

### ISSUE THREAD

**Issue by natecavanaugh** [eslint/eslint#5384]
Hi there,
This seems to be related to #5295 
Using the following rule config:
```
indent: [2, "tab"]
```
The following lines will throw errors:
```js
instance.layoutSerializer = new LayoutSerializer(
	{
		builder: instance.get('formBuilder')
	}
);
```

However, with the same config, the following will **not** throw any errors:
```js
instance.layoutSerializer = LayoutSerializer(
	{
		builder: instance.get('formBuilder')
	}
);
```
The reason for this is because on [this line](https://github.com/eslint/eslint/commit/afb6708eff08b739b7e57b7d807cf8ef9a790b14#diff-f7ae198ea24182d178b8f3376e61b6edR434), it's not checking for `NewExpression` which is what the first block of code will register as.

**Comment by ilyavolodin** [comment]
Sounds like an oversight to me. @gyandeeps @BYK ideas?

**Comment by gyandeeps** [comment]
@natecavanaugh Do you want to send a PR for this quick fix?

**Comment by natecavanaugh** [comment]
Sure thing :)

**Comment by natecavanaugh** [comment]
Sent https://github.com/eslint/eslint/pull/5389
